package/src/components/MessageList/MessageList.tsx

```diff
@@ -784,27 +785,47 @@ const MessageListWithContext = <
   };
 
   const onScrollToIndexFailedRef = useRef<
-    FlatListProps<MessageType<StreamChatGenerics>>['onScrollToIndexFailed']
```

Comment:
why replace this typing?! its future proof

Reply:
I fully agree, yet I was not successful in extending this one with the optional `counter` arg